ssd1306/ssd1306_conf_template.h

```diff
@@ -17,6 +17,17 @@
 //#define STM32H7
 //#define STM32F7
 
+/*
+* Warning: If use DMA, you must enable I2C event interrupt or SPI global
+* interrupt, otherwise the hard fault interrupt will be triggered.
+*
+* If you use DMA and ssd1306_UpdateScreen() will be called in an interrupt
+* service routine (ISR), you must ensure the ISR can be interrupted by DMA
+* ISR for I2C or SPI, which unlocks I2C or SPI hardware, otherwise I2C or
+* SPI hardware will be busy forever.
```

Comment:
I'm afraid this comment is not good enough. Could you please provide a step-by-step instructions on how to do it in STM32CubeMX and/or corresponding code examples? Better place it in a separate Markdown file and link it from the code.

Reply:
OK